unime/src/routes/(app)/me/+page.svelte

```diff
@@ -120,18 +116,18 @@
         <!-- TODO: feature disabled: "Add self-signed credential" -->
         <!-- <ButtonRounded label="Add" icon={PlusCircle} /> -->
       </div>
-    {:else if $state?.user_journey}
+      <!-- {:else if $state?.user_journey} -->
```

Comment:
Why comment it out entirely? I know we're not using `user_journey` at the moment, but it is still part of the Ferris dev profile and can be enabled through an env variable.

Reply:
My apologies, was not aware that it is possible to run this code. Commented out to get rid of a TS error, but putting it back.